WORKSPACE

```diff
@@ -115,6 +115,30 @@ graknlabs_graql_maven_dependencies = "maven_dependencies")
 graknlabs_graql_maven_dependencies()
 
 
+##################################
+# Load Distribution dependencies #
+##################################
+
+# TODO: rename the macro we load here to deploy_github_dependencies
+load("@graknlabs_bazel_distribution//github:dependencies.bzl", "github_dependencies_for_deployment")
+github_dependencies_for_deployment()
+
+load("@graknlabs_build_tools//bazel:dependencies.bzl", "bazel_rules_docker")
+bazel_rules_docker()
+
+load("@io_bazel_rules_docker//repositories:repositories.bzl",
+bazel_rules_docker_repositories = "repositories")
+bazel_rules_docker_repositories()
+
+load("@io_bazel_rules_docker//container:container.bzl", "container_pull")
+container_pull(
+  name = "openjdk_image",
+  registry = "index.docker.io",
+  repository = "library/openjdk",
+  tag = "8"
+)
```

Comment:
same question as in https://github.com/graknlabs/graql/pull/28/files#r269271565

Why do we need these docker rules?

Reply:
fixed